### DISCUSSION TITLE
Meaning of traning input "Type"

### DISCUSSION THREAD

**Discussion by FelipeSBarros** [semiautomaticgit/SemiAutomaticClassificationPlugin#263] [Q&A] [answered]
Dear all (and @semiautomaticgit :),
I am working on a land use classification class. One point I didn't understand is the *type* each training input has.
It starts as  "R&S" type (image 1), which I believe maening "ROI and Signature", but after while (and I didn't realized when, or after which process) it is converted to "Sig" (image 2).
![image](https://user-images.githubusercontent.com/7208674/235756811-95a27f5d-3087-4c29-bcbf-2a6422292fbb.png)

![image](https://user-images.githubusercontent.com/7208674/235756706-1c5866c3-6d16-458d-b0b0-00bb4932013b.png)

Could someone explain me about it and its implications in the clasification process?
Best regards

Felipe

**Comment by semiautomaticgit** [comment, marked as answer]
Hello @FelipeSBarros ,

probably the scp file got corrupted and the ROI polygons were lost.
R&S means that both polygons and signatures are present in the training input.
Only S means that the polygon is missing for that signature.

In case you have a backup file (SCP creates automatically a backup file of the training input) you could import the ROI polygons.

Best regards
